.github/workflows/build.yml

```
@@ -48,45 +48,61 @@ jobs:
         with:
           name: kubo
           path: cmd/ipfs/ipfs
-  interop:
+  Helia-interop:
```

```suggestion
  helia-interop:
```

Personally don't like the capitalisation here, but I don't care much.